lms/djangoapps/ora_staff_grader/views.py

```diff
@@ -0,0 +1,86 @@
+"""
+Views for Enhanced Staff Grader
+"""
+from edx_rest_framework_extensions.auth.jwt.authentication import JwtAuthentication
+from edx_rest_framework_extensions.auth.session.authentication import SessionAuthenticationAllowInactiveUser
+from opaque_keys.edx.keys import UsageKey
+from rest_framework.generics import RetrieveAPIView
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from xmodule.modulestore.django import modulestore
+
+from lms.djangoapps.ora_staff_grader.serializers import (
+    CourseMetadataSerializer,
+    OpenResponseMetadataSerializer,
+    SubmissionMetadataSerializer
+)
+from lms.djangoapps.ora_staff_grader.utils import call_xblock_json_handler
+from openedx.core.djangoapps.content.course_overviews.api import get_course_overview_or_none
+from openedx.core.lib.api.authentication import BearerAuthenticationAllowInactiveUser
+
+
+class InitializeView(RetrieveAPIView):
+    """
+    GET course metadata
+
+    Response: {
+        courseMetadata
+        oraMetadata
+        submissions
+    }
+
+    Returns:
+        200
+        400
+        403
+        404
+        405
+    """
+    authentication_classes = (
+        JwtAuthentication,
+        BearerAuthenticationAllowInactiveUser,
+        SessionAuthenticationAllowInactiveUser,
+    )
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request, *args, **kwargs):
+        ora_location = request.query_params['ora_location']
+
+        # Get ORA block
+        ora_usage_key = UsageKey.from_string(ora_location)
+        openassessment_block = modulestore().get_item(ora_usage_key)
+
+        # Get course metadata
+        course_id = str(ora_usage_key.course_key)
+        course_metadata = get_course_overview_or_none(course_id)
+
+        # Get list of submissions for this ORA
+        submissions_metadata = self.get_submissions(request, ora_location)
+
+        # Get the rubric config for this ORA
+        rubric_config = self.get_rubric_config(request, ora_location)
+
+        return Response({
+            'courseMetadata': CourseMetadataSerializer(course_metadata).data,
+            'oraMetadata': OpenResponseMetadataSerializer(openassessment_block).data,
+            'submissions': {
+                submission_id: SubmissionMetadataSerializer(submission).data
+                for (submission_id, submission) in submissions_metadata.items()
+            },
+            'rubricConfig': rubric_config,
```

Comment:
Let's toss a ticket in the backlog to Serializer this. It's not necessary, but Serializers are a great way to improve enpdoint readability.

Reply:
Yeah, wasn't sure how to compose all of those together. Otherwise would have done that 😓

Reply:
> Good stuff. Unit tests? If we're not doing them here we should have a ticket for them

https://openedx.atlassian.net/browse/AU-349